CHANGES.txt

```diff
@@ -18,6 +18,9 @@ Breaking Changes
 Changes
 =======
 
+ - Added support for hash sums (MD5, SHA1, SHA256). Please
```

Comment:
Please change to: **Implemented hash sum scalar functions (MD5, SHA1, SHA256).**